openpdf/src/main/java/com/lowagie/text/Document.java

```diff
@@ -106,120 +111,121 @@ public class Document implements DocListener {
+            if (input != null) {
+                Properties prop = new Properties();
+                prop.load(input);
+                releaseVersion = prop.getProperty("bundleVersion", releaseVersion);
+            }
+        } catch (IOException ignored) {
+            // ignore this and leave the default
+        }
+        return releaseVersion;
+    }
 
     /**
-     * The DocListener.
+     * Allows the pdf documents to be produced without compression for debugging
+     * purposes.
      */
-    private final List<DocListener> listeners = new ArrayList<>();
+    public static boolean compress = true;
+
     /**
-     * Is the document open or not?
+     * When true the file access is not done through a memory mapped file. Use it if the file
+     * is too big to be mapped in your address space.
      */
+    public static boolean plainRandomAccess = false;
+
+    /** Scales the WMF font size. The default value is 0.86. */
+    public static float wmfFontCorrection = 0.86f;
+
+    /** The DocListener. */
+    private List<DocListener> listeners = new ArrayList<>();
+
+    /** Is the document open or not? */
     protected boolean open;
-    /**
-     * Has the document already been closed?
-     */
+
+    /** Has the document already been closed? */
     protected boolean close;
-    /**
-     * The size of the page.
-     */
-    protected Rectangle pageSize;
 
     // membervariables concerning the layout
-    /**
-     * margin in x direction starting from the left
-     */
+
+    /** The size of the page. */
+    protected Rectangle pageSize;
+
+    /** margin in x direction starting from the left */
     protected float marginLeft = 0;
-    /**
-     * margin in x direction starting from the right
-     */
+
+    /** margin in x direction starting from the right */
     protected float marginRight = 0;
-    /**
-     * margin in y direction starting from the top
-     */
+
+    /** margin in y direction starting from the top */
     protected float marginTop = 0;
-    /**
-     * margin in y direction starting from the bottom
-     */
+
+    /** margin in y direction starting from the bottom */
     protected float marginBottom = 0;
-    /**
-     * mirroring of the left/right margins
-     */
+
+    /** mirroring of the left/right margins */
     protected boolean marginMirroring = false;
+
     /**
      * mirroring of the top/bottom margins
-     *
-     * @since 2.1.6
+     * @since    2.1.6
      */
     protected boolean marginMirroringTopBottom = false;
-    /**
-     * Content of JavaScript onLoad function
-     */
+
+    /** Content of JavaScript onLoad function */
     protected String javaScript_onLoad = null;
-    /**
-     * Content of JavaScript onUnLoad function
-     */
+
+    /** Content of JavaScript onUnLoad function */
     protected String javaScript_onUnLoad = null;
-    /**
-     * Style class in HTML body tag
-     */
+
+    /** Style class in HTML body tag */
```

Comment:
The code was reformatted recently, can you try using the configuration in the `config`-directory, so the formatting is not changed. There are 2 configuration files there:

- eclipse-java-openpdf-style.xml
- intellij-java-openpdf-style.xml